packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -417,6 +417,18 @@ class FileCollection extends DocumentCollection {
     return this.stackClient.fetchJSON('POST', uri`/files/trash/${id}`)
   }
 
+  /**
+   * Copy a file.
+   *
+   * @param {string} id   - The file's id
+   * @returns {Promise}   - A promise that returns the copied file if resolved.
+   * @throws {FetchError}
+   *
+   */
+  async copy(id) {
+    return await this.stackClient.fetchJSON('POST', `/files/${id}/copy`)
+  }
```

Comment:
The route can take more parameters than just this `id`. See https://docs.cozy.io/en/cozy-stack/files/#post-filesfile-idcopy 

Since this part of CozyClient is to consume the stack's APIs, can you add this capabilities?

Reply:
Updated!